### ISSUE TITLE
Format Code is not changing when .vscode/settings.json is changed

### ISSUE THREAD

**Issue by brennanMKE** [Microsoft/vscode#4037]
When I change values in `settings.json` it does not change the behavior of Format Code immediately. I have to quit and restart VS Code to change the behavior. 

### Testing

If I change the tab size to a value like 6 and format a source file it does not change. I restart VS Code and try again and I get 6 spaces for indentation. I believe I was able to change `settings.json` in a previous version and it would change immediately. I did upgrade to the latest version this week. There appear to be some regressions. Love the code folding though! Thanks!

```json
{
    "editor.tabSize": 2,
    "editor.insertSpaces": true,
    "editor.wrappingIndent": "same"
}
```

**Comment by alexandrudima** [comment]
@brennanMKE What language are you formatting in?

**Comment by brennanMKE** [comment]
TypeScript

**Comment by dbaeumer** [comment]
This is fixed for 0.10.12

**Comment by brennanMKE** [comment]
Great, thanks.
